client/client.go

```diff
@@ -801,26 +783,37 @@ func (r *NotaryRepository) Update(forWrite bool) error {
 // Returns a tufclient.Client for the remote server, which may not be actually
 // operational (if the URL is invalid but a root.json is cached).
 func (r *NotaryRepository) bootstrapClient(checkInitialized bool) (*tufclient.Client, error) {
-	var (
-		rootJSON   []byte
-		err        error
-		signedRoot *data.SignedRoot
-	)
+	version := 0
```

Comment:
Also cc: @endophage  - unless I'm missing something, it seems that we can be stricter about version checks and enforce that all versions are >= 1 right?